registration/src/main/java/org/spine3/samples/lobby/registration/order/OrderAggregate.java

```diff
@@ -0,0 +1,391 @@
+    }
+
+    @Assign
+    public List<Message> handle(MarkSeatsAsReserved command, CommandContext context) {
+        Validator.checkNotConfirmed(getState(), command);
+        Validator.validateCommand(command);
+
+        final ImmutableList.Builder<Message> result = ImmutableList.builder();
+        final List<SeatQuantity> reservedSeats = command.getSeatList();
+
+        if (isOrderPartiallyReserved(reservedSeats)) {
+            final OrderPartiallyReserved partiallyReserved = EventBuilder.buildOrderPartiallyReserved(command);
+            result.add(partiallyReserved);
+
+            final OrderTotalsCalculated newTotalsCalculated = buildOrderTotalsCalculated(command.getOrderId(),
+                    getState().getConferenceId(), command.getSeatList());
+            result.add(newTotalsCalculated);
+        } else {
+            final OrderReservationCompleted reservationCompleted = EventBuilder.buildOrderReservationCompleted(command);
+            result.add(reservationCompleted);
+        }
+        return result.build();
+    }
+
+    @Assign
+    public OrderExpired handle(RejectOrder command, CommandContext context) {
+        Validator.checkNotConfirmed(getState(), command);
+        Validator.validateCommand(command);
+        final OrderExpired result = OrderExpired.newBuilder()
+                .setOrderId(command.getOrderId())
+                .build();
+        return result;
+    }
+
+    @Assign
+    public OrderConfirmed handle(ConfirmOrder command, CommandContext context) {
+        Validator.checkNotConfirmed(getState(), command);
+        Validator.validateCommand(command);
+        final OrderConfirmed result = OrderConfirmed.newBuilder()
+                .setOrderId(command.getOrderId())
+                .build();
+        return result;
+    }
+
+    @Assign
+    public OrderRegistrantAssigned handle(AssignRegistrantDetails command, CommandContext context) {
+        Validator.validateCommand(command);
+        final OrderRegistrantAssigned result = OrderRegistrantAssigned.newBuilder()
+                .setOrderId(command.getOrderId())
+                .setPersonalInfo(command.getRegistrant())
+                .build();
+        return result;
+    }
+
+    @Apply
+    private void apply(OrderPlaced event) {
+        final Order.Builder state = Order.newBuilder()
+                .setId(event.getOrderId())
+                .setConferenceId(event.getConferenceId())
+                .addAllSeat(event.getSeatList());
+        incrementState(state.build());
+    }
+
+    @Apply
+    private void apply(OrderUpdated event) {
+        updateSeats(event.getSeatList());
+    }
+
+    @Apply
+    private void apply(OrderPartiallyReserved event) {
+        updateSeats(event.getSeatList());
+    }
+
+    @Apply
+    private void apply(OrderReservationCompleted event) {
+        updateSeats(event.getSeatList());
+    }
+
+    @Apply
+    private void apply(OrderTotalsCalculated event) {
+        // NOP
```

Comment:
We have to create `apply` methods because Spine throws an exception if there is no applier for an event that command handler generated.